composer.json

```diff
@@ -18,7 +19,9 @@
         "sentry/sentry-laravel": "^1.0",
         "spatie/laravel-backup": "^5.11",
         "spatie/laravel-cors": "^1.3",
-        "spatie/laravel-permission": "^2.28"
+        "spatie/laravel-permission": "^2.28",
+        "theseer/tokenizer": "^1.2",
```

Comment:
These were dependencies from some composer update? They were auto included?

Reply:
Yes, it did.